### ISSUE TITLE
Group multiple files

### ISSUE THREAD

**Issue by contrapunctus-1** [iNPUTmice/Conversations#4225]
If somebody has sent multiple files (e.g. images), it would be easier to scroll past them, and also cleaner-looking, if they were grouped into a single 'message', a la WhatsApp or Signal. The group would show some thumbnails for the files, and tapping the group would list the files so they can then be opened.

**Comment by licaon-kter** [comment]
This sounds like it's tied to a "included gallery" that it's not included at the moment, also a known/closed issue

**Comment by Yuubi-san** [comment]
@licaon-kter
> This sounds like it's tied to a "included gallery"

Not necessarily. This can be just regular bubble collapsing, with thumbnails from multiple consecutive messages displayed in a single bubble side by side, and maybe in multiple rows.

**Comment by ericschdt** [comment]
> This sounds like it's tied to a "included gallery" that it's not included at the moment, also a known/closed issue

At least an `Image Viewer` was on the [roadmap to 3.0 from 12 Sep 2020](https://nitter.net/pic/media%2FEhsj-QbWAAEYH19.jpg%3Fname%3Dorig).

**Comment by licaon-kter** [comment]
All roadmap items might be on hold until the "rewrite" I think.

**Comment by contrapunctus-1** [comment]
> This sounds like it's tied to a "included gallery" that it's not included at the moment, also a known/closed issue

The kind of view I had in mind, which would appear when the group of files is tapped, is already present in Conversations - the same as when one taps "View Media" in a chat's details.
